src/components/Layout/Layout.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import '../../containers/App.css'
+import {NavLink} from 'react-router-dom';
+import { signout } from '../../store/reducers/signinSlice';
+
+const Layout = (props) => {
+
+  const renderWelcome = () => 
+    props.signInSlice.submitted
+      ? <li><span>Welcome <span style={{fontWeight: 'bold'}}>{props.signInSlice.username}</span></span></li>
+      : null
+    
+  const renderSettings = () => 
+    (props.signInSlice.submitted && props.signInSlice.userrole === 'Admin')
```

Comment:
Place 'Admin' and 'User' into global constants, and use these constants instead of hardcoded values

Reply:
done